src/sensors/SensorFusion.cpp

```diff
@@ -173,9 +173,9 @@ namespace SlimeVR
         
         void SensorFusion::calcLinearAcc(const sensor_real_t accin[3], const sensor_real_t gravVec[3], sensor_real_t accout[3])
         {
-            accout[0] = accin[0] - gravVec[0] * CONST_EARTH_GRAVITY;
-            accout[1] = accin[1] - gravVec[1] * CONST_EARTH_GRAVITY;
-            accout[2] = accin[2] - gravVec[2] * CONST_EARTH_GRAVITY;
+            accout[0] = (accin[0] - gravVec[0]) * CONST_EARTH_GRAVITY;
+            accout[1] = (accin[1] - gravVec[1]) * CONST_EARTH_GRAVITY;
+            accout[2] = (accin[2] - gravVec[2]) * CONST_EARTH_GRAVITY;
```

Comment:
Breaking change for every IMU other than BNO.

Vector `accin` is in m/s^2 and `gravVec` is a unit vector, if you have issues with linear acceleration in another IMU implementation, perhaps you're passing acceleration into fusion in G?

Reply:
I did not realize I was using this wrong, I have reverted my changes and will go about this another way in the IMU specific code. I will probably change the units the IMU uses.